app/routes/articles/ArticleEditRoute.ts

```diff
@@ -23,7 +24,7 @@ const mapStateToProps = (state, props) => {
   });
 
   const currentUser = selectCurrentUser(state);
-  const authors = article?.authors?.length
+  const authors: Array<DetailedUser> = article?.authors?.length
```

Comment:
Please be consistent on this
```suggestion
  const authors: DetailedUser[] = article?.authors?.length
```
It's not _super_ important since it's equivalent, but still nice to be consistent